dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -10,335 +10,98 @@ nav_section: prebid-amp
-
-In the page's body, implement the `draw3p` function as shown below.  This implementation should work even on pages that have multiple AMP ad types.
-
-{: .alert.alert-danger :}
-Don't forget to add the required arrays at the end of the function showing the ad networks used and the domains allowed to load this file in an iFrame.  You **must** add all `amp-ad` types on page to the `draw3p` array, even if they are not being used with Prebid demand.
-
-For more information about `draw3p`, see:
-
-+ [Its implementation](https://github.com/ampproject/amphtml/blob/e5501a30adf15c8fef049729f5e0e3137dbb18ca/3p/integration.js#L252)
-
-+ [The issue that led to its implementation](https://github.com/ampproject/amphtml/issues/1210)
-
-```javascript
-      /** The draw3p function is the integration point between AMP and content in third party
-        * iframes. For more info see: https://github.com/ampproject/amphtml/blob/e5501a30adf15c8fef049729f5e0e3137dbb18ca/3p/integration.js#L252
-        */
-      draw3p(function(config, done) {
-        if(config.prebid) {
-          if (typeof window.context.master.adCalls === 'undefined') {
-            window.context.master.adCalls = [];
-          }
-            if (window.context && window.context.isMaster && config.prebid) {
-                var prebid = config.prebid;
-                adUnits = prebid.adUnits;
-                requestBidsDuration = prebid.requestBidsDuration;
-                loadPrebidJS();
-            }
-            window.context.master.adCalls.push(setTargeting.bind(null, config, done));
-        }
-        else{
-          done(config);
-        }
-
-      }, ['doubleclick', 'taboola'], ['vip.local', 'preprod.metro.co.uk', 'preprod.metro.co.uk', 'metro.co.uk']);
-// the first array contains ad networks used, the second domains that allow us to load this file in an iframe
-```
-
-
-#### 7. Listen to the ad request from the creative
-
-In the same `script` tag in the body as the previous step, call
-
-```javascript
-listenAdRequestFromCreative();
-```
+    <amp-ad width="300" height="250"
+            type="doubleclick"
+            data-slot="/19968336/universal_creative"
+            rtc-config='{"vendors": {"PrebidAppNexus": {"PLACEMENT_ID": "12345679"}}, "timeoutMillis": 500}'>
+    </amp-ad>
 
-which is one of the helper functions defined in Step 3.
+{% endhighlight %}
 
 ### HTML Creative
 
-This is the creative that your Ad Ops team uploads to the ad server.  You can see it in the AMP example in our repo: [creative.html](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/amp/creative.html).
+This is the creative that your Ad Ops team needs to upload to the ad server (it's also documented at [Setting up Prebid for AMP in DFP]({{site.github.url}}/adops/setting-up-prebid-for-amp-in-dfp.html)).
 
-It basically does two things:
+It works by calling out to the cache on a [Prebid Server][PBS] instance, which will handle the communication via [AMP RTC][RTC-Overview].
```

Comment:
This description is very vague.  If we're going to describe how it works, let's be specific.

Reply:
Removed this line after discussing it with @mkendall07